services/ymax-planner/src/gas-estimation.ts

```diff
@@ -0,0 +1,81 @@
+import type { AxelarChain } from '@agoric/portfolio-api/src/constants';
+import { gasLimitEstimates } from './support.ts';
+
+const AGORIC_CHAIN = 'agoric';
+const BLD_TOKEN = 'ubld';
+
+export const prepareGasEstimator =
+  ({
+    axelarApiAddress,
+  }: {
+    axelarApiAddress: string;
+  }) =>
+  (chainName: AxelarChain) => {
+    const axelarEstimateGasUrl = `${axelarApiAddress}/gmp/estimateGasFee`;
```

Comment:
The `axelarApiAddress` values in main.ts already have trailing slashes (e.g., "https://api.axelarscan.io/").
```suggestion
    const axelarEstimateGasUrl = `${axelarApiAddress.replace(/\/+$/, '')}/gmp/estimateGasFee`;
```